Readme.md

```diff
@@ -1,70 +1,121 @@
-# Power BI for HTML / JavaScript
+# powerbi-client
+
+## Installation
+
+Install via Nuget:
 
-## Install via Nuget
 `Install-Package Microsoft.PowerBI.JavaScript -Pre`
 
-## Install from NPM
-`npm install powerbi-client`
+Install from NPM:
+
+`npm install -save powerbi-client`
+
+Install from Bower:
+
+`bower install powerbi-client --save`
 
-## Install from Bower
-`bower install powerbi-client`
+Installing beta versions:
+
+`npm install --save powerbi-client@beta`
 
 ## Setup Power BI for embedding
-Add the Power BI script include before your apps closing `</body>` tag
+
+Add the Power BI script before your apps closing `</body>` tag
 
 `<script src="/bower_components/powerbi-client/dist/powerbi.js"></script>`
 
-Add your access token to your app so it can be accessed.  
-*This can be set statically or dynamically via JavaScript.*
+This exposes two globals `Powerbi` which is the service class and `powerbi` which is an instance of the service.
 
+
+# Embedding
+
+## PaaS
+### When using PaaS model the tokens issued are for a specific report or tile and the token should be associated with the embed url on the same element to ensure each has a unique token allows embeding multiple reports using the same service instance.
+
+```
+<div
+	powerbi-type="report"
+	powerbi-embed-url="https://embedded.powerbi.com/appTokenReportEmbed?reportId=5dac7a4a-4452-46b3-99f6-a25915e0fe55"
+	powerbi-access-token="eyJ0eXAiO...Qron7qYpY9MI"
+></div>
+```
+
+## SaaS
+### When using SaaS model the tokens issued are for a specific user who can view many types of visuals. This means you can add this as a global token reused for all embeded visuals as shown below:
 ```
 <script>
-	window.powerbi = window.powerbi || {};
 	window.powerbi.accessToken = '{{AccessToken}}';
 </script>
+...
+
+<div
+	powerbi-type="report"
+	powerbi-embed-url="https://embedded.powerbi.com/appTokenReportEmbed?reportId=5dac7a4a-4452-46b3-99f6-a25915e0fe55"
```

Comment:
I would update this url to show an example of a sass report embed url.  This one is specific to paas app token embed.